ibu-imager/seedcreator/seedcreator.go

```diff
@@ -241,39 +243,36 @@ func (s *SeedCreator) createContainerList() error {
 	return nil
 }
 
-func (s *SeedCreator) backupSeedCertificates(ctx context.Context) error {
-	s.log.Info("Backing up seed cluster certificates for recert tool")
-	if err := os.MkdirAll(common.BackupCertsDir, os.ModePerm); err != nil {
-		return fmt.Errorf("error creating %s: %w", common.BackupCertsDir, err)
+func BackupCertificates(ctx context.Context, client *clusterinfo.InfoClient, certDir string) error {
```

Comment:
@leo8a @tsorya 
Any suggestion on where to put this function so it can be used in LCA controller?
Can't put it in utils since it will cause import cycle.
`InfoClient` is importing both utils.

Reply:
utils sounds good just need to move GetConfigMapData to utils too, not sure why those functions are part of info client.
We can actually create some client_helper interface or just file in utils and move there all the functions that wrap client operations